assets/js/components/notifications/AdBlockerWarningMessage.js

```diff
@@ -44,6 +44,7 @@ export default function AdBlockerWarningMessage( {
 
 	return (
 		<Notice
+			className="googlesitekit-notice--small"
```

Comment:
Let's revisit this in a follow up to control this with a prop rather than passing a notice-specific class here.